frontend/packages/console-app/src/components/resource-quota/ResourceQuotaCharts.tsx

```diff
@@ -12,10 +12,14 @@ type ResourceQuotaChartsProps = {
 const ResourceQuotaCharts = ({ resourceQuota }: ResourceQuotaChartsProps): JSX.Element => {
   const { t } = useTranslation();
   const charts = Object.keys(resourceQuota.status?.hard ?? {}).map((resourceName) => {
-    const hard = resourceQuota.status.hard[resourceName];
-    const used = resourceQuota.status.used?.[resourceName];
+    const hard = resourceQuota.status?.hard?.[resourceName || ''] ?? '';
+    const used = resourceQuota.status?.used?.[resourceName || ''] ?? '';
```

Comment:
These default values can be removed if the `used` and `hard` props optional in [getUsedPercentage](https://github.com/openshift/console/blob/main/frontend/packages/console-app/src/components/resource-quota/utils.ts#L16) and [getLabelAndUsage](https://github.com/openshift/console/blob/main/frontend/packages/console-app/src/components/resource-quota/utils.ts#L29)